pxtcompiler/emitter/emitter.ts

```diff
@@ -526,10 +542,10 @@ namespace ts.pxtc {
         return isClassType(t)
     }
 
-    function arrayElementType(t: Type): Type {
+    function arrayElementType(t: Type, idx = -1): Type {
```

Comment:
idx is not used?

Reply:
yeah, in future this should look if it's a tuple type and return the type for specific index; doesn't seem needed right now though